webpack/translations-loader.js

```diff
@@ -22,6 +22,7 @@
  *   app.steps.click: 'this button'
  * }
  */
+/* eslint-disable array-callback-return */
```

Comment:
note: I think it is a little bit risky to refactor this function. The issue is the function using map as loop to go through all the key/value pair.. Just ignore this function only from the lint error array-callback-return in order to add lint as part of the GA